### PR TITLE
#1555 Show a bigger logo in the landing page

### PR DESCRIPTION
**Pull request by junkiattan**
[#1555](https://github.com/PowerPointLabs/PowerPointLabs/issues/1555)

As discussed with @damithc, 

1. I have moved the logo in the navigation bar in by 42px to increased focus on the 'PowerPointLabs' logo
2. I have also reduced margin above the "PowerPoint supercharged" to reduce whitespace and improve visibility of the landing site

### CODE REVIEW

**Comment by damithc** [comment]
Can give a screenshot?

**Comment by junkiattan** [comment]
@damithc 

The original landing site
capture

New landing site
capture1

**Comment by junkiattan** [comment]
@damithc ,

In this 3rd picture I have increased left padding to 50px and made the logo abit bigger than before to make it look more prominent, if you prefer this over the 2nd one I can push the changes. Currently its at the 2nd shot

capture

**Comment by damithc** [comment]
Looks fine. 👍

**Comment by junkiattan** [comment]
@damithc 

Can I merge the pull request in? Or will I have to wait for further reviews?

**Comment by damithc** [comment]
My review is just for the screenshot. One of the senior devs need to review/merge the code.

File: _includes/navbar.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
-  <div class="navbar-header pull-left">
+  <div class="navbar-logo navbar-header pull-left">
     <a class="navbar-brand" href="{{ site.baseurl }}/">
-      <img src="{{ site.baseurl }}/img/Logo.png">
+      <img width="auto" height="26" src="{{ site.baseurl }}/img/Logo.png">
```
Comment (Skaty) [inline _includes/navbar.html]: I don't think `width="auto"` is valid HTML. The attribute only takes a value (in pixels), according to the relevant MDN documentation.

**Review by Skaty — CHANGES_REQUESTED** [review]
Aside from the inline comments:
- Remove the `.vs` files, as it's your editor's config files.

File: _includes/navbar.html (hunk 1 above)
Comment (junkiattan) [reply _includes/navbar.html]: @Skaty 

It seemed that I didnt need to specify the width, I have removed that and also the .vs files ;)

File: css/main.less [hunk 2]
```diff
@@ -63,7 +63,6 @@ h1, h2, h3, h4, h5, h6 {
   color: @primary-brand;
   font-weight: 700;
   margin-bottom: 0.5em;
-  margin-top: 2em
```
Comment (Skaty) [inline css/main.less:66]: Why is this line removed? Will this affect the styles of other headers (i.e. elements with header tags) on the website?

**Review by Skaty — CHANGES_REQUESTED** [review]
Clarification required.

File: css/main.less [hunk 3]
```diff
@@ -58,12 +58,16 @@ p, a, li {
...
+    margin-top: 0em;
+}
+
 h1, h2, h3, h4, h5, h6 {
-  font-family:'Lato', sans-serif;
-  color: @primary-brand;
-  font-weight: 700;
-  margin-bottom: 0.5em;
-  margin-top: 2em
+    font-family: 'Lato', sans-serif;
```
Comment (Skaty) [inline css/main.less]: According to [§2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation) of the Google HTML/CSS Style Guide (as adopted in [process](/oss-generic/process)), 2 spaces should be used as indentation.

File: css/main.less [hunk 4]
```diff
@@ -58,12 +58,16 @@ p, a, li {
   color: @wet-asphalt;
 }
 
+.landingPage-title {
```
Comment (Skaty) [inline css/main.less]: This violates [the CSS class naming convention](https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Css.adoc#classes).

**Review by Skaty — CHANGES_REQUESTED** [review]
Changes requested: coding style violations.

**Review by Skaty — APPROVED** [review]
LGTM. :+1:

**Comment by Skaty** [comment]
@junkiattan FYI: For subsequent PRs, it's best if you don't merge it on your own after approval. Let the main reviewer or senior devs merge it.

**Comment by junkiattan** [comment]
@Skaty 

Apologies on my part. I will remember for subsequent PRs;)

**Pull request merged**